crates/e2e/tests/e2e/univ2.rs

```diff
@@ -116,7 +116,7 @@ async fn test(web3: Web3) {
             // solver participated in the competition
             && data.participants.iter().any(|p| p.participant.0 == solver.address().0)
             // and won the auction
-            && data.reference_scores.first().is_some_and(|score| score.solver.0 == solver.address().0)
+                    && data.reference_scores[0].solver.0 == solver.address().0
```

Comment:
The original should be kept since that cannot panic. Also the new version has a wrong indentation.